ext/AvizExt/viz/clustering.jl

```diff
@@ -32,8 +40,8 @@ function ADRIA.viz.ts_cluster!(g::Union{GridLayout,GridPosition}, data::Abstract
         axis_opts...
     )
 
-    # Filter clusters and data for non-zero clusters
-    clusters_filtered = filter(c -> c != 0, clusters)
+    # Filter off clusters with index -1  (those represent constant series)

```

Comment:
```suggestion
    # Filter clusters with index value of "-1"  (these represent constant series)

```

Reply:
Actually, the code does exactly what you describe so not sure if this comment is needed?